FirebaseInAppMessaging/Sources/Public/FirebaseInAppMessaging/FIRInAppMessaging.h

```diff
@@ -14,6 +14,9 @@
  * limitations under the License.
  */
 
+#import <TargetConditionals.h>
+#if TARGET_OS_IOS
+
```

Comment:
I'm good with this now but I think we should discuss how we can make this a better experience for developers by [marking our APIs](https://developer.apple.com/documentation/swift/objective-c_and_c_code_customization/marking_api_availability_in_objective-c) with `API_AVAILABLE(ios(10))` and the like. That way they get nice messages with compiler errors instead of "missing symbol".

Reply:
I'll experiment with this.

Reply:
Actually it should be `API_UNAVAILABLE(macos,tvos,watchos)`.  Added a post-beta task to the project to revisit this.